src/randoop/instrument/CoveredClassVisitor.java

```diff
@@ -0,0 +1,82 @@
+package randoop.instrument;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Set;
+
+import randoop.ExecutionVisitor;
+import randoop.sequence.ExecutableSequence;
+
+/**
+ * A {@link ExecutionVisitor} that polls a set of coverage instrumented
+ * classes and adds each covered class to an {@link ExecutableSequence} after
+ * it is executed.
+ */
+public class CoveredClassVisitor implements ExecutionVisitor {
+
+  /** the classes to be polled */
```

Comment:
Minor:  Please capitalize the first letter.  I would add a period at the end too, though that isn't required if it's just a sentence fragment.